### PR TITLE
output message to delete the weblinks installation manually

### PR DESCRIPTION
**Pull request by compojoom**
On Windows if you kill the robo script when the repository is being cloned, then a file in .git\objects\pack get's locked (most probably by git.exe).

When we try to delete tests/joomla-cms, we are no longer able to do this. The only possible way to then clone would be to manually delete the folder and run robo after that.

### CODE REVIEW

**Comment by compojoom** [comment]
Actually this shouldn't be a windows only problem and you should be able to reproduce it on a mac. When you run robo run:tests and the script starts cloning kill the script when you see the loading percents - like at 20%. Then run the test again. Most probably it will not be able to delete some files, because they are locked.

**Comment by therunnergit** [comment]
I consider this a more generic case, there's a lot of reasons why you may not be able to delete the folder. I was able to reproduce on Linux by purposefully setting different permissions to the folder, and I can confirm your code provides a correct information message.
@Test works fine.

**Comment by therunnergit** [comment]
@compojoom can you please fix conflicts by replacing `'tests/joomla-cms3'` with `$this->cmsPath` in two places?

File: RoboFile.php [hunk 1]
```diff
@@ -178,6 +178,16 @@ public function createTestingSite($use_htaccess = false)
...
 		if (is_dir($this->cmsPath))
 		{
+			try
+			{
+				$this->taskDeleteDir($this->cmsPath)->run();
+			}
+			catch (Exception $e)
+			{
+				// Sorry, we tried :(
+				$this->say('Sorry, you will have to delete ' . $this->cmsPath). ' manually. ');
```
Comment (Kubik-Rubik) [inline RoboFile.php]: You got a syntax error in line 188 -> remove ) please, then I can merge it. Thanks!

**Comment by compojoom** [comment]
closing in favor of https://github.com/joomla-extensions/weblinks/pull/131

**Pull request closed without merging**